contracts/TwabRewards.sol

```diff
@@ -169,8 +169,9 @@ contract TwabRewards is ITwabRewards {
 
         uint256 _rewardsAmount;
         uint256 _userClaimedEpochs = _claimedEpochs[_promotionId][_user];
+        uint256 _epochIdsLength = _epochIds.length;
```

Comment:
This is funny...it's a calldata array.  How much gas does this actually save?

Reply:
I'd say 3 in gas per iteration since a CALLDATALOAD consume 3 gas.